Update project-lifecycle.md (#351)
ossf/tac: process/project-lifecycle.md

### Archived

> Open source projects have a lifecycle and there are times when projects become inactive due to a variety of reasons. There are also cases where a project may no longer want to be supported by the OpenSSF, the given effort no longer has broad community interest and particpation, or the OpenSSF TAC may no longer wish to recommend the use of a project. Archiving happens through a vote of the TAC, and can be requested by the corresponding project's lead(s) or a TAC member.  TI's that are dormant, with no activity for 9 months (meetings, mailing lists, or other publicly viewable channels) in a row should be considered good candidates for Archiving.

Reviewer: ```suggestion
Open source projects have a lifecycle and there are times when projects become inactive due to a variety of reasons. There are also cases where a project may no longer want to be supported by the OpenSSF, the given effort no longer has broad community interest and participation, or the OpenSSF TAC may no longer wish to recommend the use of a project. Archiving happens through a vote of the TAC, and can be requested by the corresponding project's lead(s) or a TAC member.  TI's that are dormant, with no activity for 9 months (meetings, mailing lists, or other publicly viewable channels) in a row should be considered good candidates for Archiving.
```

* If the project's inactivity or inconsistent maintenance presents a user security risk.
* If the project's design or approach is no longer a recommended best practice.

#### Considerations when Archiving a TI that has sub-efforts embedded within ####

1.) Overseeing TI will be moving to "Archived" status

> 2.) Active subgroups (SIG or project) is still active and desires to continue collaborating

Reviewer 2: Nit

```suggestion
2.) Whether subgroups/-TIs (SIG or project) are still active and desire to continue collaborating
```

3.) Subgroup reviews active TIs to see if there is alignment of vision and purpose (https://openssf.org/community/openssf-working-groups/)
4.) Subgroup meets with TI leader and group to discuss adoption
5.) New TI agrees to become new oversight group for subgroup
6.) PRs filed in TI and TAC repos to note change of status

> 7.) Archiving TI has PR filed noting new status

Reviewer 2: I like this list, but I might split this into steps/tasks for the hosting TI vs. the sub-TI that wants to transition. I also think that'll also make it clearer about who's responsible for which of these tasks (TAC vs. TI chairs, vs sub-TIs).

> 1.) Overseeing TI will be moving to "Archived" status
> 2.) Whether subgroups/-TIs (SIG or project) are still active and desire to continue collaborating
> 3.) Subgroup reviews active TIs to see if there is alignment of vision and purpose (https://openssf.org/community/openssf-working-groups/)
> 4.) Subgroup meets with TI leader and group to discuss adoption
> 5.) New TI agrees to become new oversight group for subgroup
> 6.) PRs filed in TI and TAC repos to note change of status
> 7.) Archiving TI has PR filed noting new status

Reviewer 3: ```suggestion
1. Overseeing TI will be moving to "Archived" status
2. Whether subgroups/-TIs (SIG or project) are still active and desire to continue collaborating
3. Subgroup reviews active TIs to see if there is alignment of vision and purpose (https://openssf.org/community/openssf-working-groups/)
4. Subgroup meets with TI leader and group to discuss adoption
5. New TI agrees to become new oversight group for subgroup
6. PRs filed in TI and TAC repos to note change of status
7. Archiving TI has PR filed noting new status
```

Minor: GitHub's Markdown renderer will smoosh all these items onto one line if the format `1.)` is used, but `1.` will create a list 🤷

Reviewer 4: In addition to removing the `)`, every item can be `1.` and the markdown render will automatically increment. That makes future PRs easier if anything ever gets added or removed from the middle of the list.
